com.github.iortcw.yaml

```diff
@@ -0,0 +1,85 @@
+app-id: com.github.iortcw
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "20.08"
+command: iortcw_sp.sh
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --persist=.wolf
+
+cleanup:
+- /app/include
+- /app/lib/*.a
+- /app/lib/*.la
+- /app/lib/pkgconfig
+
+modules:
+
+- name: iortcw-source
+  buildsystem: simple
+  build-commands:
+   # For historic reasons, single- and multiplayer are different binaries
+   - cd /run/build/iortcw-source/SP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make copyfiles
+   - cd /run/build/iortcw-source/MP/ && BUILD_GAME_QVM=1 COPYDIR=/app/bin/ make copyfiles
+  sources:
+  - type: archive
+    url: https://github.com/iortcw/iortcw/archive/1.51c.tar.gz
+    sha256: 0f6c136db1e5c6115e044190655163b9784bbef9b0667ec2a8900abd79565e83
+    
+- name: iortcw-assets
+  buildsystem: simple
+  build-commands:
+   - ls -la
+   - cp ./main/mp_bin0.pk3 /app/bin/main/
+   - cp ./main/mp_rend2_shaders0.pk3 /app/bin/main/
+   - cp ./main/sp_rend2_shaders0.pk3 /app/bin/main/
+  sources:
+  - type: archive
+    url: https://github.com/iortcw/iortcw/releases/download/1.51c/iortcw-1.51c-linux-x86_64.zip
+    sha256: b322dcfd997c5e3e5a5b3f86d1b3525190b1fb5aee36fe4afa65a01b7cec845e
+
+- name: launcher
+  buildsystem: simple
+  sources:
+  # singleplayer
+  - type: script
+    dest-filename: iortcw_sp.sh
+    commands:
+    - ls ~/.wolf/main/pak0.pk3 || zenity --error --text "Game data (pak0.pk3 etc.) must be added to:\n~/.var/app/com.github.iortcw/.wolf/main/" --ok-label "Close" --width=400
+    - /app/bin/iowolfsp.x86_64
+    # multiplayer
+  - type: script
+    dest-filename: iortcw_mp.sh
+    commands:
+    - ls ~/.wolf/main/pak0.pk3 || zenity --error --text "Game data (pak0.pk3 etc.) must be added to:\n~/.var/app/com.github.iortcw/.wolf/main/" --ok-label "Close" --width=400
+    - /app/bin/iowolfmp.x86_64
+  - type: file
+    path: com.github.iortcw.sp.desktop
+  - type: file
+    path: com.github.iortcw.mp.desktop
+  - type: file
+    path: com.github.iortcw.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Totally forgot about that